src/client/cli/formatter/format_utils.cpp

```diff
@@ -110,3 +110,19 @@ void mp::format::filter_aliases(google::protobuf::RepeatedPtrField<multipass::Fi
             aliases.DeleteSubrange(i, 1);
     }
 }
+
+mp::FormatUtils::FormatUtils(const Singleton<FormatUtils>::PrivatePass& pass) noexcept
+    : Singleton<FormatUtils>::Singleton{pass}
+{
+}
+
+std::string mp::FormatUtils::convert_to_localtime(const google::protobuf::Timestamp& timestamp) const
+{
+    std::ostringstream oss;
+    std::time_t t = google::protobuf::util::TimeUtil::TimestampToTimeT(timestamp);
+
+    // ISO 8601: YYYY-MM-DDThh:mm:ss±hhmm
+    oss << std::put_time(std::localtime(&t), "%FT%T%z");
```

Comment:
Currently have this set up to print out the UTC offset. Excluding `±hhmm` looks a bit more presentable, but then means that the datetime is ambiguous on whether it defines UTC time or localtime. See [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601#Local_time_(unqualified)).